.github/workflows/database-restore.yml

```diff
@@ -34,12 +33,34 @@ jobs:
          INSTALL_CONDUIT: true
 
       - name: Backup Teacher Training API Prod Database
-        run: cf conduit teacher-training-api-postgres-prod -- pg_dump --encoding utf8 --clean --no-owner --if-exists -f prod_backup.sql
-      
+        run: |
+          now=$(date +"%S_%M_%H_%m_%d_%Y")
+          prod_sql_file=prod_backup-$now.sql
+          touch $prod_sql_file
+          cf conduit teacher-training-api-postgres-prod -- pg_dump --encoding utf8 --clean --no-owner --if-exists -f $prod_sql_file
+
+      - uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS_PRODUCTION }}
+
+      - uses: Azure/get-keyvault-secrets@v1.0
+        with:
+          keyvault: "s121p01-shared-kv-01"
+          secrets: 'TTAPI-STORAGE-ACCOUNT-CONNECTION-STRING-PRODUCTION'
+        id: myGetSecretAction
+
+      - uses: bacongobbler/azure-blob-storage-upload@v1.1.1
+        with:
+          source_dir: "./"
+          container_name: 'sqldbdump'
+          connection_string: ${{ steps.myGetSecretAction.outputs.TTAPI-STORAGE-ACCOUNT-CONNECTION-STRING-PRODUCTION }}
```

Comment:
change to `getKeyVaultSecret` or similar?
remove `my` :)

Reply:
lol